src/actions/photos.js

```diff
@@ -0,0 +1,106 @@
+/**
+  Photos related features
+**/
+
+import cozy from 'cozy-client-js'
+
+import {
+  COZY_PHOTOS_DIR_ID,
+  FETCH_PHOTOS,
+  FETCH_PHOTOS_FAILURE,
+  RECEIVE_PHOTOS,
+  UPLOAD_PHOTOS,
+  UPLOAD_PHOTOS_SUCCESS,
+  UPLOAD_PHOTOS_SUCCESS_WITH_CONFLICTS,
+  UPLOAD_PHOTOS_FAILURE
+} from './constants'
```

Comment:
@gregorylegarec and @goldoraf, I can change it then. Just for saying, @gregorylegarec even if it's in same files as the related action creator file, it will be in a global context because it's used by the related reducer file and the UI reducer file. Also, it simplified a lot [these import lines](https://github.com/cozy/cozy-photos-v3/blob/Add_redux_main_features/src/reducers/ui.js#L3) in the UI reducers which need all constants from actions creators.